src/client/client4.js

```diff
@@ -583,6 +583,13 @@ export default class Client4 {
         );
     };
 
+    revokeAllSessionsForUser = async (userId) => {
+        return this.doFetch(
+            `${this.getUserRoute(userId)}/sessions/revoke/all`,
+            {method: 'post', body: JSON.stringify({user_id: userId})}
```

Comment:
We don't actually need the user_id in the request body, correct?

Reply:
Gah, of course we don't. I'll clean up these few issues and re-submit.